examples/examplesAccount/main.go

```diff
@@ -15,7 +16,7 @@ func main() {
 	ep := blockchain.NewElrondProxy(examples.TestnetGateway, nil)
 
 	// Retrieving network configuration parameters
-	networkConfig, err := ep.GetNetworkConfig()
+	networkConfig, err := ep.GetNetworkConfig(context.Background())
```

Comment:
If I understood correctly: you only need a specific timeout for certain endpoints, and for the API calls that don't need a timeout, you simply pass `context.Background()`?

Reply:
exactly: context.Background() will never close the context.Done() channel. Since all these are examples and we extensively used the standard context.Context interface, any integrator can implement the timeout mechanism, if required.